bin/publib-pypi

```diff
@@ -65,17 +72,11 @@ if [ -n "${PYPI_TRUSTED_PUBLISHER:-}" ]; then
   
   export TWINE_USERNAME="__token__"
   export TWINE_PASSWORD="$api_token"
-fi
-
-if [ -z "${PYPI_DISABLE_ATTESTATIONS:-}" ]; then
-  echo "Signing packages with pypi-attestations"
-  python3 -m pypi_attestations sign *
-fi
 
-# Build upload command with optional attestations
-upload_opts="--verbose --skip-existing"
-if [ -z "${PYPI_DISABLE_ATTESTATIONS:-}" ]; then
-  upload_opts="$upload_opts --attestations"
+  if [ -z "${PYPI_DISABLE_ATTESTATIONS:-}" ]; then
+    echo "Signing packages with pypi-attestations"
+    python3 -m pypi_attestations sign *
```

Comment:
As far as I understand, the attestation should be disabled in my github action because it does not set id-token permission. However, the workflow generated by projen cdk-construct-library does not seem to set PYPI_DISABLE_ATTESTATIONS environment variable. Not sure if this works.